src/commands/codepush/deployment/list.ts

```diff
@@ -107,7 +112,11 @@ export default class CodePushDeploymentListListCommand extends AppCommand {
   }
 
   private async generateMetricsJSON(deployment: models.Deployment, client: AppCenterClient): Promise<models.CodePushReleaseMetric> {
-    const metrics: models.CodePushReleaseMetric[] = await this.getMetrics(deployment, client);
+    let metrics: models.CodePushReleaseMetric[] = [];
+    if (!this.disableDeploymentMetric) {
+      out.text("Note: To Disbale Deployment Metrics Info set -m | --disableDeploymentMetric option to true");
```

Comment:
I don't think we need to display such message. If you want to improve the UX in case the command fails due to exceeding the limit, we can print something like "You can try disabling fetching metrics to reduce the number of sending requests" only when the command fails with that specific error.
In normal case there is no reason for such message.

Reply:
made modification in the error message for rate limiting